### PR TITLE
Add Head

### PR DESCRIPTION
**Pull request by TheElementGuy**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -1 +1,26 @@
-
+<!doctype html>
+<html>
+	<head>
+		<meta charset="UTF-8" />
```
Comment (HatchlingsCheeseburger) [inline index.html:4]: I thought it was "utf-8" not "UTF-8".

Comment (TheElementGuy) [reply index.html:4]: Both are the same thing.

File: index.html [hunk 2]
```diff
@@ -1 +1,26 @@
...
+		<meta name="viewport" content="width=device-width,initial-scale=1" />
```
Comment (HatchlingsCheeseburger) [inline index.html:5]: What is this?

File: index.html [hunk 3]
```diff
@@ -1 +1,26 @@
...
+		<link rel="stylesheet" href="https://www.w3schools.com/w3css/4/w3.css" />
```
Comment (HatchlingsCheeseburger) [inline index.html:6]: What is this?

File: index.html (hunk 2 above)
Comment (TheElementGuy) [reply index.html:5]: That makes it so that "50%" for example is 50% for whatever device not just a chromebook.

File: index.html (hunk 3 above)
Comment (TheElementGuy) [reply index.html:6]: Adding W3.CSS

File: index.html [hunk 4]
```diff
@@ -1 +1,26 @@
...
+		<style>
+
```
Comment (HatchlingsCheeseburger) [inline index.html:8]: Why did you add the `style` if you don't even use CSS?

File: index.html (hunk 1 above)
Comment (HatchlingsCheeseburger) [reply index.html:4]: ok.

File: index.html (hunk 2 above)
Comment (HatchlingsCheeseburger) [reply index.html:5]: ???

File: index.html (hunk 4 above)
Comment (TheElementGuy) [reply index.html:8]: We might use some and it's nice to have the tags.

File: index.html (hunk 3 above)
Comment (HatchlingsCheeseburger) [reply index.html:6]: no w3.css

Comment (HatchlingsCheeseburger) [reply index.html:6]: `not in my opinion`

Comment (HatchlingsCheeseburger) [reply index.html:6]: `
do the stupid
`

Comment (TheElementGuy) [reply index.html:6]: Well W3.css makes it look good on all devices you know.

Comment (HatchlingsCheeseburger) [reply index.html:6]: ### Why I don't

> I will not be forced

Comment (TheElementGuy) [reply index.html:6]: OK no w3.css.

Comment (HatchlingsCheeseburger) [reply index.html:6]: - [x] make it w3-css
- [x] only do `Pull requests`
